### ISSUE TITLE
Win 10 Game DVR bar behaves incorrectly when DS 4 is wireless.

### ISSUE THREAD

**Issue by artlp** [Jays2Kings/DS4Windows#178]
If I am playing any game and open the gamebar with Win+G (or guide/PS button) the buttons on it will infinitely scroll quickly as if I am holding TAB button. This glitch doesn't happen in wired mode.

**Comment by Nemurikami** [comment]
Same problem. Happens too in MS Store, scrolls to the bottom and won't stop. First noticed the issue while playing Skyrim (before remaster releases ;) because it was stuttering. Worked the problem down to my controller but tried a second PS4 controller and the same thing happened. Definitely stopped happening as soon as I shutdown DS4Windows.

I'll try using a USB cable to wire in as a temporary solution. Guessing MS updated something that broke something in the DS4Windows code. Might be in preparations for the PS4 dongle for PS4 Now coming out soon.

**Comment by Nemurikami** [comment]
Double checked the new downloads page after last post,..

Literally said, "Posted 5 minutes ago"...
https://github.com/Jays2Kings/DS4Windows/releases/tag/v.1.4.52

Well, damn.

Thank you! Hope this fixes it!

**Comment by Jays2Kings** [comment]
I'm not certain this build will fix this, I'll try to see why this happens in future updates.

**Comment by Nemurikami** [comment]
Yeah, I tested it and it unfortunately did not fix this issue on my rig. I had intended to post an update-reply but fell asleep, sorry  m(-__-)m

The related issues I've seen,
-- both my controllers are showing >30ms latency
-- the Xbox pop-up menu scrolls infinitely between choices
-- the MS Store scrolls to the bottom continuously - like it'll force-scroll down as I try to scroll up - stops when I exit DS4W
-- Skyrim is stuttering badly like from some input trying to yank attention from the game's IRQ handling system
-- looking at sensor readings in DS4W, my gyro looks off... like the middle one is cranked hard right (both controllers)

I'll try to grab some screen clips.

Thank U for Ur quick response and attentiveness!  w(^_____^)w!!

**Comment by Nemurikami** [comment]
Unloading, reloading, killing drivers, updating drivers, switching Bluetooth and USB,... trying to trick it into working properly one way or another...

But fear it's got something to do with this:

http://www.bestbuy.com/site/sony-dualshock4-usb-wireless-adapter/5589800.p?skuId=5589800

**Comment by Nemurikami** [comment]
The adaptor helped stablize my Bluetooth signal, but I still need DS4W to make it work with my game. DS4Win! Still staying awesome classy. W(^__^)W!!
